kotlin/lib/src/main/kotlin/goodmetrics/Metrics.kt

```diff
@@ -1,5 +1,22 @@
 package goodmetrics
 
+sealed interface SupportedDimensionType
+data class StringDimension(val value: String) : SupportedDimensionType {
+    override fun toString(): String {
+        return value
+    }

```

Comment:
I overrode these to ensure the intended `value` is written as expected

Reply:
seems reasonable, idk if it's really necessary